WebLibs/api/IDAS.js

```diff
@@ -42,16 +47,20 @@ export class IDAS {
         }
     }
 
-    async authenticateWithJwt() {
+    async authenticateWithJwt(authPath) {
         if (!authJwtToken) {
+            const authEndpoint = (new URL(window.location.href).origin) + authPath;
+            let authUrlCallback = `${authEndpoint}?r=%target%&t=%jwt%`;
+            authUrlCallback = authUrlCallback.replace('%target%', encodeURIComponent(window.location.href));
+
             const url = new URL(apiBaseUrl);
             url.pathname = "/Session";
```

Comment:
@gdl-reif: What do you think about using `controller/action` API pattern in new code?

I'm not talking about replacing it in all places, but only about using it in new code. This will help .NET 6 WebAPI a lot

Reply:
Makes sense IMO - yes!